katdal/chunkstore.py

```diff
@@ -0,0 +1,219 @@
+################################################################################
```

Comment:
Do you want to call it `chunk_store.py`? PEP8 is a bit vague ("Underscores can be used in the module name if it improves readability.") so it's up to your taste.

Reply:
I've got both: `lazy_indexer.py`, `concatdata.py`, `visdatav4.py`, etc.

So I'll leave as is.